tests/python/gaia-ui-tests/gaiatest/apps/settings/regions/addons.py

```diff
@@ -33,8 +44,3 @@ def tap_first_item(self):
 
     def toggle_addon_status(self):
```

Comment:
yes, I'm looking forward to fix my script regarding the toggle/switch once the bug 1202388 lands